deployments/dockerfiles/gosec/Dockerfile

```diff
@@ -1,11 +1,17 @@
-# Dockerfile used to create "husyci/gosec" image
-# https://hub.docker.com/r/huskyci/gosec/
```

Comment:
Could you keep this `# https://hub.docker.com/r/huskyci/gosec/` as well? 🙃

Reply:
Sorry about that, I'd looked only at your comment, not the code... fixed